### PR TITLE
chore: release v6.10.3

### PR DESCRIPTION
**Pull request by shortcuts**
This pull request prepares the following release:
| Repository | Branch | Update | Change |
|---|---|---|---|
| [react-instantsearch](https://github.com/algolia/react-instantsearch) | releases/v6.10.3 | patch | [`6.10.2` → `6.10.3`](https://github.com/algolia/react-instantsearch/compare/v6.10.2...releases/v6.10.3) |

### Release Summary
This is going to be published with the following command:

| Dir | Command |
|---|---|
| packages/react-instantsearch-core | npm_config_registry=https://registry.npmjs.org/ npm publish --tag latest |
| packages/react-instantsearch-dom-maps | npm_config_registry=https://registry.npmjs.org/ npm publish --tag latest |
| packages/react-instantsearch-dom | npm_config_registry=https://registry.npmjs.org/ npm publish --tag latest |
| packages/react-instantsearch-native | npm_config_registry=https://registry.npmjs.org/ npm publish --tag latest |
| packages/react-instantsearch | npm_config_registry=https://registry.npmjs.org/ npm publish --tag latest |

### Merging Instructions
When merging this pull request, you need to **Squash and merge** and make sure that the title starts with `chore: release v6.10.3`.
<details>
<summary>See details</summary>

After that, a commit `chore: release v6.10.3` will be added and you or your CI can run `shipjs trigger` to trigger the release based on the commit.
Squash and merge
</details>

---

_This pull request is automatically generated by [Ship.js](https://github.com/algolia/shipjs)_.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## [6.10.3](https://github.com/algolia/react-instantsearch/compare/v6.10.2...v6.10.3) (2021-03-03)
+
+
```
Comment (Haroenv) [inline CHANGELOG.md:3]: ### Bug Fixes

* **RelevantSort:** Rename `SmartSort` widget to `RelevantSort` ([#3026](https://github.com/algolia/react-instantsearch/issues/3026))

**Review by Haroenv — APPROVED** [review]

**Comment by eunjae-lee** [comment]
@shortcuts can you take a look why tests are failing?

**Comment by Haroenv** [comment]
very odd that ci is failing here, but not on master, it seemingly has SmartSort in some places?

**Comment by shortcuts** [comment]
> very odd that ci is failing here, but not on master, it seemingly has SmartSort in some places?

It seems that it's not really up to date with master even though it says it is

File: yarn.lock [hunk 2]
```diff
@@ -16782,6 +16782,27 @@ react-inspector@^2.3.0, react-inspector@^2.3.1:
...
+  resolved "https://registry.yarnpkg.com/react-instantsearch-dom/-/react-instantsearch-dom-6.10.2.tgz#c969f84243fed8ddd9eebe440042a3dd32ef5c86"
+  integrity sha512-5u5t5lvydgTq218JmbOv7BZqFmvGcpgQTEi/02M/T0BLjToDu8bMItpXXQHwWNlOidJl/M+iphybxkxAvJukFQ==
+  dependencies:
+    "@babel/runtime" "^7.1.2"
+    algoliasearch-helper "^3.4.3"
+    classnames "^2.2.5"
+    prop-types "^15.6.2"
+    react-fast-compare "^3.0.0"
+    react-instantsearch-core "^6.10.2"
+
```
Comment (eunjae-lee) [inline yarn.lock:16805]: this seems wrong?

Comment (shortcuts) [reply yarn.lock:16805]: Yep, I don't have it in master

Comment (shortcuts) [reply yarn.lock:16805]: It was removed here: https://github.com/algolia/react-instantsearch/commit/fa815f970dea19f96c8275a1c08061f34c4a50ad#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deL16785-L16805, that's it?

Comment (eunjae-lee) [reply yarn.lock:16805]: hmm it's because of examples using the old version

**Pull request closed without merging**